Codecademy/gamut
# feat(Card): Add outline shadow variant

### Overview


We need to add this [“multi” layer card look on path cards](https://www.figma.com/file/rGzb9m1sPiFihBX3u8WEmG/Catalog-revamp?node-id=828%3A46841) that then transitions on hover. [This is how the hover effect should work](https://www.figma.com/proto/rGzb9m1sPiFihBX3u8WEmG/Catalog-revamp?page-id=801%3A42740&node-id=867%3A49918&viewport=236%2C114%2C0.31&scaling=min-zoom) (ignore text animation for now).

Approved by Tamar and Mark ✅


### PR Checklist

- [x] Related to designs: https://www.figma.com/file/rGzb9m1sPiFihBX3u8WEmG/Catalog-revamp?node-id=828%3A46841
- [x] Related to JIRA ticket: [DISC-921]

#### Testing Instructions

1. Visit https://63601dfd14a48b1de22e9d1c--gamut-preview.netlify.app/?path=/docs/atoms-card--card#shadow-variants
2. See new shadow variant section with new outline variant 
3. Hover over to see effect
4. Visit mono pr env /catalog page to see effect on career cards ([must be whitelisted in experiment](https://app.optimizely.com/v2/projects/21056540090/experiments/22405500170/whitelist))

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/33301)  | This variant will only be on portal pages |
| Monorepo       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/511)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-511)   |




[DISC-921]: https://codecademy.atlassian.net/browse/DISC-921?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
